tests/claws-utils/utils-test.cpp

```diff
@@ -0,0 +1,10 @@
+//
+// Created by doom on 28/07/18.
+//
+
+#include <gtest/gtest.h>
```

Comment:
You should try to include some claws library utilities and look if everything compile

Reply:
It does